Mathlib/Topology/Instances/ENNReal.lean

```diff
@@ -283,6 +283,20 @@ protected theorem tendsto_nhds {f : Filter α} {u : α → ℝ≥0∞} {a : ℝ
   simp only [nhds_of_ne_top ha, tendsto_iInf, tendsto_principal]
 #align ennreal.tendsto_nhds ENNReal.tendsto_nhds
 
+theorem ofReal_cinfi (f : α → ℝ) [Nonempty α] :
```

Comment:
The place of this lemma is not ideal. You put this between `tendsto_nhds` and `tendsto_nhds_zero`, which are two very related lemmas. Could you put it further down, where it would not break a sequence of related results?

Reply:
moved it to the end of the TopologicalSpace section.